h/auth/tokens.py

```diff
@@ -11,15 +11,15 @@
 
 @implementer(IAuthenticationToken)
 class Token(object):
-    """
+    r"""
     A long-lived API token for a user.
 
     This is a wrapper class that wraps an ``h.models.Token`` and provides an
     implementation of the ``IAuthenticationToken`` interface.
 
     Unlike ``models.Token`` this class is not a sqlalchemy ORM class so it can
     be used after the request's db session has been committed or invalidated
-    without getting ``DetachedInstanceError``\ s from sqlalchemy.
+    without getting DetachedInstanceError\ s from sqlalchemy.
```

Comment:
What was the `\` for originally? Since DetachedInstanceError refers to an entity it does need to be quoted for Sphinx doesn't it?

Reply:
Oops. You need a space after double backticks in rst, otherwise the double backticks won't be recognised as a code tag. But you don't want a literal space (you don't want a space between the `DetachedInstanceError` and the s) so you have to use a `\ ` which inserts a space that does not appear in the output. Test here: <http://rst.ninjs.org/#YGBEZXRhY2hlZEluc3RhbmNlRXJyb3JgYHMKCmBgRGV0YWNoZWRJbnN0YW5jZUVycm9yYGAgcwoKYGBEZXRhY2hlZEluc3RhbmNlRXJyb3JgYFwgcw==>

But I see that I accidentally deleted the backticks. I've put those back now.